zuul.d/project.yaml

```diff
@@ -44,6 +44,11 @@
             files:
               - playbooks/ansible-network-appliance-base/.*
               - playbooks/ansible-network-vyos-appliance/.*
+        - ansible-security-integration-qradar-python38:
+            files:
+              - playbooks/ansible-test-network-integration-base/files/bootstrap-qradar.yaml
+              - playbooks/ansible-network-appliance-base/.*
+              - zuul.d/ansible-security-jobs.yaml
```

Comment:
there is no need to match on this file, zuul does it by default

Reply:
ok, interesting.